wsproto/handshake.py

```diff
@@ -429,22 +438,24 @@ def server_extensions_handshake(
         for extension in supported:
             if extension.name == name:
                 accept = extension.accept(offer)
-                if accept is True:
-                    accepts[extension.name] = True
-                elif accept is not False and accept is not None:
-                    accepts[extension.name] = accept.encode("ascii")  # type: ignore
+                if isinstance(accept, bool):
```

Comment:
Does `isinstance(accept, bool) and accept` work here?

Reply:
No, since then it will fallback to the `elif accept is not None` branch, but what should happen is to do nothing.